apps/studio/components/interfaces/Storage/StoragePolicies/StoragePolicies.tsx

```diff
@@ -78,6 +88,50 @@ export const StoragePolicies = () => {
   // Policies under storage.buckets
   const storageBucketPolicies = filter(policies, { table: 'buckets' })
 
+  /**
+   * Filter buckets based on search string
+   * - Filter buckets by name matching the search string
+   * - Show all policies for filtered buckets (policies are not filtered)
```

Comment:
As in, don’t take the policy name as a search query. Figured bucket name is enough to go from.